pkg/ebpf/c/tracee.bpf.c

```diff
@@ -5133,6 +5133,31 @@ int BPF_KPROBE(trace_security_task_setrlimit)
     return events_perf_submit(&p, 0);
 }
 
+SEC("kprobe/ptrace")
```

Comment:
Not sure if there is a raw tracepoint for this, but i'm pretty sure that even a regular tracepoint has better performance than a kprobe.

Reply:
According to @yanivagman in #2085, where I suggested something similar with a regular tracepoint, there should be an option for a raw tracepoint.

Reply:
If you succeed at doing a raw_tracepoint on a syscall that's indeed preferred. Let me know if that's the case.
tracepoint requires debugfs which is not currently a requirement of tracee.